### PR TITLE
[build] Switch to NS 2.0 and .NET 4.7.1

### PR DESCRIPTION
**Pull request by garuma**
To be merged when we are ready to do the jump elsewhere

### CODE REVIEW

File: src/Microsoft.Language.Xml.Editor/Microsoft.Language.Xml.Editor.csproj [hunk 1]
```diff
@@ -1,18 +1,18 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <AssemblyName>Microsoft.Language.Xml.Editor</AssemblyName>
-    <TargetFramework>net46</TargetFramework>
+    <TargetFramework>net461</TargetFramework>
```
Comment (decriptor) [inline src/Microsoft.Language.Xml.Editor/Microsoft.Language.Xml.Editor.csproj]: Should this be `net471`?

**Comment by kzu** [comment]
If this code is intended to ship and run under VS2017, please keep this in mind: https://github.com/github/VisualStudio/issues/1849#issuecomment-411570902

IOW, it should target net46 and no higher (or multi-target maybe? so dev16 can use ns2?). 

Not sure which .NET will be the one that runs dev16, but I'm guessing it's likely it will be 471 or maybe higher by the time it ships.

**Comment by garuma** [comment]
This is purely for Dev16 which will be NS2 compatible

**Comment by kzu** [comment]
Great!

**Pull request merged**